server/src/controllers/auth/register.ts

```diff
@@ -12,7 +12,7 @@ const UserRegisterSchema = z.object({
   password: z.string(),
   email: z.string().email().toLowerCase(),
   last_name: z.string(),
-  pic_square: z.string(),
+  pic_square: z.string(), // TODO: WHYYYYYY?? Should not be from user input thats fucking mental
```

Comment:
@EvenBiggerManGhark - hahahaha I guess we didn't sort out this horrible ToDo

Reply:
hahahah Another days work